tutorials/noisepy_scoped_download_s3_store.ipynb

```diff
@@ -0,0 +1,251 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Download S3-hosted Noisepy Data\n",
+    "\n",
+    "This notebook is designed to query cross-corelations data calculated by noisepy, hosted on S3, and downloaded locally.\n",
+    "\n",
+    "This notebook assumes that you have installed the noisepy package. It installs Python tools for MongoDB, queries our SCOPED data base, and parse the S3-hosted data into the ASDF H5 data format."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: pymongo in /Users/marinedenolle/miniconda3/envs/noisepy/lib/python3.10/site-packages (4.6.1)\n",
+      "Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /Users/marinedenolle/miniconda3/envs/noisepy/lib/python3.10/site-packages (from pymongo) (2.4.2)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install pymongo"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Pinged your deployment. You successfully connected to MongoDB!\n"
+     ]
+    }
+   ],
+   "source": [
+    "from pymongo.mongo_client import MongoClient\n",
+    "from pymongo.server_api import ServerApi\n",
+    "\n",
+    "uri = \"mongodb+srv://user:hG0osHjBdXovsxq8@scoped.nfcjw38.mongodb.net/?retryWrites=true&w=majority\"\n",
+    "\n",
+    "# Create a new client and connect to the server\n",
+    "client = MongoClient(uri, server_api=ServerApi('1'))\n",
+    "\n",
+    "# Send a ping to confirm a successful connection\n",
+    "try:\n",
+    "    client.admin.command('ping')\n",
+    "    print(\"Pinged your deployment. You successfully connected to MongoDB!\")\n",
+    "except Exception as e:\n",
+    "    print(e)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
```

Comment:
Please clear all the cell outputs. Jupyter Notebook: `Edit -> Clear All Outputs`, or in VS Code:

<img width="503" alt="image" src="https://github.com/noisepy/NoisePy/assets/10456546/84d52655-8828-4f75-a3a1-6993e4194e35">

Reply:
cleared up